website/templates/state_biz.html

```diff
@@ -6,7 +6,6 @@ <h1 class="font-weight title_blm">Black Businesses to Support in {{title}}</h1>
 
```

Comment:
Idk why that happens I havent seen that before on my machine. and no it doesnt add any more data onto the site. I think if we have pagination it would be great.